daemon/volumes_linux_unit_test.go

```diff
@@ -6,15 +6,16 @@ import (
 	"testing"
 
 	"github.com/docker/docker/runconfig"
-	"github.com/docker/docker/volume"
-	"github.com/docker/docker/volume/drivers"
+	volumedriver "github.com/docker/docker/volume/drivers"
 )
 
 type fakeDriver struct{}
 
-func (fakeDriver) Name() string                                                      { return "fake" }
-func (fakeDriver) Create(name string, opts map[string]string) (volume.Volume, error) { return nil, nil }
-func (fakeDriver) Remove(v volume.Volume) error                                      { return nil }
+func (fakeDriver) New(string, interface{}) volumedriver.Driver     { return nil }
+func (fakeDriver) Name() string                                    { return "fake" }
+func (fakeDriver) Create(name string) (volumedriver.Volume, error) { return nil, nil }
+func (fakeDriver) Remove(v volumedriver.Volume) error              { return nil }
+func (fakeDriver) GetVolume(string) volumedriver.Volume            { return nil }
```

Comment:
I just noticed that the naming of the test file was outdated as well.  Can you elaborate on the initial comment though?